### DISCUSSION TITLE
Schématisations

### DISCUSSION THREAD

**Discussion by ClodianeC** [ClodianeC/error404#29] [General]
La schématisation pour la page liste des artistes
![SchématisationListeArtiste_CharetteC](https://user-images.githubusercontent.com/98475192/197571850-31726adf-5970-46c7-9ca6-9c14a50b7488.jpg)

**Comment by rosaliecsfoy** [comment]
La schématisation pour la fiche d'un artiste:
![schématisation-rosalie](https://user-images.githubusercontent.com/92270297/197860203-12d93be2-b362-4617-ba4f-783901d49a30.png)

*PS: À noter que j'ai coupé le haut de page ainsi que le pied de page, car il sera fait qu'une seule fois et qu'il n'est pas dans mon mandat*

**Comment by isaacdube** [comment]
![Schéatisation](https://user-images.githubusercontent.com/98491693/197879919-5a90f35b-1b40-4ab4-9467-897595dbfffc.png)
